osu.Android/osu.Android.csproj

```diff
@@ -1,73 +1,22 @@
+    <TargetFramework>net6.0-android</TargetFramework>
+    <OutputType>Exe</OutputType>
     <RootNamespace>osu.Android</RootNamespace>
     <AssemblyName>osu.Android</AssemblyName>
-    <AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
-    <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a</AndroidSupportedAbis>
-    <EnableLLVM>false</EnableLLVM> <!-- This currently causes random lockups during gameplay. https://github.com/mono/mono/issues/18973 -->
+
+    <!-- This currently causes random lockups during gameplay. https://github.com/mono/mono/issues/18973 -->
+    <EnableLLVM>false</EnableLLVM>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <MandroidI18n>cjk;mideast;other;rare;west</MandroidI18n>
-    <AndroidDexTool>d8</AndroidDexTool>
-    <AndroidLinkTool>r8</AndroidLinkTool>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <AndroidLinkMode>None</AndroidLinkMode>
-    <MandroidI18n>cjk;mideast;other;rare;west</MandroidI18n>
-    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="AndroidJoystickSettings.cs" />
-    <Compile Include="AndroidMouseSettings.cs" />
-    <Compile Include="GameplayScreenRotationLocker.cs" />
-    <Compile Include="OsuGameActivity.cs" />
-    <Compile Include="OsuGameAndroid.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="Properties\AndroidManifest.xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\osu.Game.Rulesets.Catch\osu.Game.Rulesets.Catch.csproj">
-      <Project>{58f6c80c-1253-4a0e-a465-b8c85ebeadf3}</Project>
-      <Name>osu.Game.Rulesets.Catch</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\osu.Game.Rulesets.Mania\osu.Game.Rulesets.Mania.csproj">
-      <Project>{48f4582b-7687-4621-9cbe-5c24197cb536}</Project>
-      <Name>osu.Game.Rulesets.Mania</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\osu.Game.Rulesets.Osu\osu.Game.Rulesets.Osu.csproj">
-      <Project>{c92a607b-1fdd-4954-9f92-03ff547d9080}</Project>
-      <Name>osu.Game.Rulesets.Osu</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\osu.Game.Rulesets.Taiko\osu.Game.Rulesets.Taiko.csproj">
-      <Project>{f167e17a-7de6-4af5-b920-a5112296c695}</Project>
-      <Name>osu.Game.Rulesets.Taiko</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\osu.Game\osu.Game.csproj">
-      <Project>{2a66dd92-adb1-4994-89e2-c94e04acda0d}</Project>
-      <Name>osu.Game</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <AndroidResource Include="Resources\drawable\lazer.png" />
-  </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.Formats.Asn1">
-      <Version>5.0.0</Version>
-    </PackageReference>
+    <PackageReference Include="Xamarin.Essentials" Version="1.7.3" />
   </ItemGroup>
```

Comment:
I'm not entirely sure whether `Xamarin.Essentials` works on Android or not (it doesn't on iOS), but it should probably be replaced with [Microsoft's variant](https://www.nuget.org/packages/Microsoft.Maui.Essentials.Runtime.android) for better compatibility with the framework.

Reply:
> I'm not entirely sure whether `Xamarin.Essentials` works on Android or not (it doesn't on iOS)

It works on my device.



> for better compatibility with the framework

Agreed, and probably we can use that on iOS too.

Reply:
The transition to Maui essentials is very straitforward. It increases required workload on develop machine (`android` -> `android-maui`).
The output apk shouldn't increase since trimming is enabled by default. I haven't confirmed though.

Reply:
@frenzibyte want to make the change here in line with your PR? probably best to have both on the same page..